virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -0,0 +1,29 @@
+[id="virt-creating-and-using-boot-sources"]
+= Creating and using boot sources
+include::modules/virt-document-attributes.adoc[]
+:context: virt-creating-and-using-boot-sources
+toc::[]
+
+A _boot source_ contains a bootable operating system (OS) and all of the configuration settings for the OS, such as drivers.
+
+You use a boot source to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.
+
+To use boot sources, you must install the latest version of {VirtProductName}.
```

Comment:
Ordinarily I would say "yes", but to do this would cause users to ask questions like "How come the other features aren't stating this explicitly as a prereq"? I think instead I'm going to remove it entirely. I initially put it there for the 2.5 release because the feature was somewhat half-baked at that point. Now that it's fully functional, I think it's on a par with all the other new 2.6 features that are "completely functional."